languages/translations/vi.json

```diff
@@ -132,27 +132,32 @@
             "sharp left": {
                 "default": "Quẹo gắt bên trái",
                 "name": "Quẹo gắt bên trái để chạy tiếp trên {way_name}",
-                "destination": "Quẹo gắt bên trái về {destination}"
+                "destination": "Quẹo gắt bên trái về {destination}",
+                "junction_name": "Quẹo gắt bên trái ở {junction_name}"
```

Comment:
👍 I think the more natural way to say this in Vietnamese would be “Đến {junction_name} quẹo gắt bên trái”. (There are a few famous intersections in Ho Chi Minh City that would get this treatment for sure.) However, in turn-by-turn guidance, it’s more effective to say the maneuver direction before ancillary details like road or junction names, as this translation does.